lib/sequences/06_argonaut_profiles_sequence.rb

```diff
@@ -30,6 +29,24 @@ class ArgonautProfilesSequence < SequenceBase
     assert errors.empty?, "Patient did not validate against profile: #{errors.join(", ")}"
   end
 
+  test 'Patient has address',
+          'Additional Patient resource requirement' do
```

Comment:
I think this description will be put into the `url` metadata section, so if no url then pass `nil` as the second parameter, and the description as the third.